dynawo/sources/Common/DYNNumericalUtils.h

```diff
@@ -27,7 +27,7 @@ namespace DYN {
 template<typename T>
 T pow_dynawo(T a, T b) {
   T value = pow(a, b);
-  if (std::isnan(value)) {
+  if (std::isnan(value) || std::isinf(value)) {
```

Comment:
Are we sure the function exists in cpp03 ? cppreference says it exists only from cpp11

Reply:
Yes, I compile in cpp03 and it isalready used in SparseMatrix for example